client/components/item/searchItem.tsx

```diff
@@ -55,10 +59,33 @@ export const SearchItem: React.FC<Props> = ({ onSelect, placeholder }) => {
       packId,
       selectedItem,
     };
-    // @ts-expect-error
-    dispatch(selectItemsGlobal(data));
+    try {
+      addGlobalItemToPack.parse(data);
+      // @ts-expect-error
+      dispatch(selectItemsGlobal(data));
+    } catch (error) {
+      const errorObject = JSON.parse(error.message);
+      const errors = {};
+      errorObject.forEach((err) => {
+        const path = err.path[0];
+        const message = err.message;
+        errors[path] = message;
+      });
+      setFormErrors(errors);
+    }
   };
 
+  if (formErrors) {
+    Object.entries(formErrors).map(([key, error]) => {
+      InformUser({
+        title: key + ' ' + error,
```

Comment:
- [ ] this is crashing lint

Reply:
i did not add this but i believe its not useful